interface/frontend/annotate_and_segment.html

```diff
@@ -7,16 +7,112 @@
+                   data-toggle="collapse"
+                   data-parent="#accordion" :href="'#collapse_' + index"
+                   :aria-controls="'collapse_' + index" :aria-expanded="index==0 ? 'true' : 'false'">
+                  {% verbatim %}
+                  <b>Nodule {{ index + 1 }}</b>
+                  {% endverbatim %}
+                </a>
+              </p>
+            </div>
+
+            <div :id="'collapse_' + index" :class="index == 0 ? 'collapse show' : 'collapse'" role="tabpanel"
+                 :aria-labelledby="'heading_' + index">
+              <div class="card-block">
+                <table>
+                  <tbody>
+                  <tr>
+                    <td>
+                      <select name="lung_orientation">
+                        <option disabled
+                                :selected="nodule.lung_orientation == {{ lung_orientations.none }} ? true : false"
+                                value> -- select an option --
+                        </option>
+                        <option :selected="nodule.lung_orientation == {{ lung_orientations.left }} ? true : false"
+                                value="left">Left lung
+                        </option>
+                        <option :selected="nodule.lung_orientation == {{ lung_orientations.right }} ? true : false"
+                                value="right">Right lung
+                        </option>
+                      </select>
+                    </td>
+                  </tr>
+                  </tbody>
+                </table>
+                <br>
+                {% csrf_token %}
+                {% verbatim %}
+                <a href="#" @click="update(nodule)">
+                  <button type="button" class="btn btn-sm btn-secondary">Accept</button>
+                </a>
+                {% endverbatim %}
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
-    </div>
+      </template>
+      <template v-else>
+        <p class="card-text">No nodules available.</p>
+      </template>
+    </div><!-- left side-->
+    <div class="col-md-8"></div><!-- right side-->
   </div>
 </div>
+
+<script src="{% static 'js/vue.js' %}"></script>
+<script src="{% static 'js/vue-resource.js' %}"></script>
+<script src="{% static 'js/tether.min.js' %}"></script>
+<script>
+  Vue.use(VueResource);
+
+  new Vue({
+
+    el: '#app',
+
+    data: {
+      nodules: [],
+    },
+
+    created: function() {
+      this.fetchNodules()
+    },
+
+    methods: {
+      fetchNodules: function() {
+        this.$http.get("{% url 'nodule-list' %}").then(
+          function(response) {
+            this.nodules = response.body;
+          },
+          function() {
+            // error callback
+          });
+      },
+      update: function(nodule) {
+        this.$http.post(nodule.url + "update",
+          { csrfmiddlewaretoken: $("input[name=csrfmiddlewaretoken]").val(),
```

Comment:
I wonder if we can do ths globally; kinda ugly to do it like this every time IMHO. jQuery has a global ajax handler- could you file a new issue?

Reply:
I filed issue #114 for that.

Reply:
Why would we use jQuery for this?

There's a simple way to set this up for Vue and Django:

```javascript
Vue.http.headers.common['X-CSRFToken']
```

Where `X-CSRFToken` is a header coming from Django.